utils/etc/ansible.cfg

```diff
@@ -19,10 +19,15 @@ nocows = 1
 
 retry_files_enabled = False
 
+deprecation_warnings=False
+
 # Need to handle:
 # inventory - derive from OO_ANSIBLE_DIRECTORY env var
 # callback_plugins - derive from pkg_resource.resource_filename
 # private_key_file - prompt if missing
 # remote_tmp - set if provided by user (cli)
 # ssh_args - set if provided by user (cli)
-# control_path
\ No newline at end of file
+# control_path
+
+stdout_callback = openshift_quick_installer
+callback_whitelist = openshift_quick_installer
```

Comment:
Are these actually needed? I was under the assumption that callbacks only needed to be whitelisted when they are disabled by  default.